validate-date-test/test.js

```diff
@@ -2,9 +2,12 @@ var test = require('tape');
 const validateDate = require('../validate-date');
 
 test('is valid date', function (t) {
-    t.plan(2);
```

Comment:
Can you check the significance of this. I don't remember exactly the reason behind putting this. Thanks

Reply:
t.plan() takes the number of tests you are going to make in the "test" session so tape knows when to stop/what to expect. I just changed it by the t.end(), which has the same function but is independent of the number of tests.

Reply:
makes sense. Thank you :)